src/role-helpers.js

```diff
@@ -138,23 +139,30 @@ function getRoles(container, {hidden = false} = {}) {
     }, {})
 }
 
-function prettyRoles(dom, {hidden}) {
+function prettyRoles(dom, {hidden, includeName}) {
   const roles = getRoles(dom, {hidden})
 
   return Object.entries(roles)
     .map(([role, elements]) => {
       const delimiterBar = '-'.repeat(50)
       const elementsString = elements
-        .map(el => prettyDOM(el.cloneNode(false)))
+        .map(
+          el =>
+            `${
+              includeName === true
+                ? `Name "${computeAccessibleName(el)}":\n`
+                : ''
+            }${prettyDOM(el.cloneNode(false))}`,
+        )
```

Comment:
Prettier makes this look funny. Why don't we do this:

```suggestion
        .map(el =>
          [
            includeName === true
              ? `Name "${computeAccessibleName(el)}":\n`
              : '',
            prettyDOM(el.cloneNode(false)),
          ].join(''),
        )
```